stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -242,6 +303,44 @@ public struct AnySequence<Element> : SequenceType {
   public func generate() -> AnyGenerator<Element> {
     return _box.generate()
   }
+  
+  public func map<T>(
```

Comment:
I think you need to copy comments from the protocol.  Swift comment parsing support does not look on other related declarations for documentation comments.